### ISSUE TITLE
Electronic Sound Experiment (demo course) not working

### ISSUE THREAD

**Issue by insad** [openedx/frontend-app-learning#939]
The Electronic Sound Experiment (demo course - week 2 / lesson 2) gives an error:

`TypeError: Cannot set properties of undefined (setting 'diagram') The tool is disabled.`

The legacy view works without problems.

**Comment by jdpark7** [comment]
I have the same problem in tutor version Quince.2

**Comment by arbrandes** [comment]
Is this a bug with the demo course or a bug with the Learning MFE?

**Comment by bradenmacdonald** [comment]
@insad @jdpark7 can any of you confirm if this is still happening in Redwood? And does it break the entire page when the bug happens? 

The source code of this MFE doesn't contain the keyword `diagram` at all so I suspect the bug is not [currently] in this MFE.

**Comment by jdpark7** [comment]
Yes. it is still happening in Redwood. The content of the Electronic Sound Experiment is shown correctly in preview mode but error occurs on live mode.
